utils/release/github-release-changelog.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Access token will be required. Check to ensure that is it provied
+if [[ -z "$GITHUB_TOKEN" ]]; then
+	echo "GITHUB_TOKEN most be set"
+fi
+auth_header="-H Authorization: Bearer $GITHUB_TOKEN"
+
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+source $script_dir/shared.sh
+
+# Get latest release version
+log "Getti previous release version tag"
+previous_tag_name=$(curl -L "https://api.github.com/repos/kuadrant/kuadrant-operator/releases/latest" -H "Accept: apllication/vnd.github+json" | yq '.tag_name')
```

Comment:
I'm not entirely sure that latest endpoint work with pre-releases, but also probably not required since the release notes might be wanted to reflect the full changes from the stable releases.

Reply:
From my testing the version that will be returned it the on marked as `latest` in the GitHub release page, not the newest entry in the release list. Currently there is a `v1.1.0-alpha1` release, but the change log would be generated comparing the changes from `v1.0.2`. For now I don't see an issue with this. Where it will run into issue is if we need to support back ports on previous minor streams.